internal/pkg/externalplugins/cherrypicker/cherrypicker_test.go

```diff
@@ -590,27 +590,26 @@ func testCherryPickPR(clients localgit.Clients, t *testing.T) {
 			branch, expectedLabels, reviewers, expectedAssignees)
 	}
 
-	if len(ghc.prs) != 2 {
-		t.Fatalf("Expected %d PRs, got %d", 2, len(ghc.prs))
+	if len(ghc.prs) != len(expectedBranches) {
+		t.Fatalf("Expected %d PRs, got %d", len(expectedBranches), len(ghc.prs))
 	}
 
-	expectedBranches := []string{"release-1.5", "release-1.6"}
+	expectedPrs := make(map[string]string)
+	for _, branch := range expectedBranches {
+		expectedPrs[expectedFn(branch)] = branch
+	}
 	seenBranches := make(map[string]struct{})
 	for _, p := range ghc.prs {
 		pr := prToString(p)
-		if pr != expectedFn("release-1.5") && pr != expectedFn("release-1.6") {
+		branch, present := expectedPrs[pr]
+		if !present {
 			t.Errorf("Unexpected PR:\n%s\nExpected to target one of the following branches: %v\n%s",
 				pr, expectedBranches, expectedFn("release-1.5"))
```

Comment:
Maybe it should be `expectedPrs` or do not ouput it.

Reply:
Good catch! Fixed.